addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -2420,10 +2420,23 @@ export function rgbToHex(rgb = '') {
             .join('')
     );
 }
-
+/**
+ * Returns position of a range in form of object (end
+ * position of a range in case of non-collapsed range).
+ *
+ * @param {HTMLElement} el element for which range postion will be calculated
+ * @param {Document} document
+ * @param {Object} [options]
+ * @param {Number} [options.marginRight] right margin to be considered
+ * @param {Number} [options.marginBottom] bottom margin to be considered
+ * @param {Number} [options.marginTop] top margin to be considered
+ * @param {Number} [options.marginLeft] left margin to be considered
+ * @param {DOMRect} [options.parentContextRect] context rect from parent
```

Comment:
I consider that expecting different keys in `options` is a signature change, albeit implicit. A customization that passes `getParentContextRect` would suddenly break with this change.

Reply:
Agreed, I did it just for documenting the `options` parameter (As you told me in the previous conversation). Should we leave documenting the parameters and only change in the docstring or am I missing something ?

Reply:
I've done the changes, can you please have a look ?